docs/DeveloperGuide.md

```diff
@@ -415,6 +434,116 @@ State: Client Window (Contact Tab)
     * 1b1. Contact is not updated and DonnaFin shows an error message. \
       Use case resumes at step 1.
 
+**UC10: Adding an Asset to a Client**
```

Comment:
I noticed you some times use Title case for 'Client' and sometimes lower case 'clients' for UC. same with other words here like 'asset', 'policy,  and  'home window'. I would not use title case for these as we're talking about nouns (and not proper nouns or class names), but up to you, as long as its consistent!

Reply:
@4d62dc3